mockserver-client.gemspec

```diff
@@ -25,16 +24,17 @@ Gem::Specification.new do |spec| # rubocop:disable Metrics/BlockLength
   spec.add_development_dependency 'bundler', '~> 1'
   spec.add_development_dependency 'rake', '~> 10.3'
   spec.add_development_dependency 'rspec', '~> 3.0'
+  spec.add_development_dependency 'rubocop', '~> 0.23'
   spec.add_development_dependency 'simplecov', '~> 0.8'
   spec.add_development_dependency 'webmock'
-  spec.add_development_dependency 'rubocop', '~> 0.23'
 
+  spec.add_dependency 'activesupport', '>= 4.1'
+  spec.add_dependency 'colorize', '~> 0.7'
   spec.add_dependency 'hashie', '~> 3.0'
-  spec.add_dependency 'json', '~> 1.8'
-  spec.add_dependency 'json_pure', '~> 1.8'
-  spec.add_dependency 'activesupport', '~> 4.1'
-  spec.add_dependency 'rest-client'
+  spec.add_dependency 'json', '>= 1.8'
+  spec.add_dependency 'activesupport', '>= 4.1'
```

Comment:
@chrisbutler I think the build here is failing b/c these dependencies were added twice:
```
spec.add_dependency 'activesupport', '>= 4.1'
spec.add_dependency 'rest-client', '>= 1.7'
```